crates/runtime/tests/models/mod.rs

```diff
@@ -332,6 +336,45 @@ async fn send_chat_completions_request(
     Ok(response)
 }
 
+/// Generic function to send a POST request, returning the response as a String.
+async fn http_post(
```

Comment:
Propose to move to mod.rs so it can be shared between different model providers / tests